### PR TITLE
A Few Collections Fixes

### PR DESCRIPTION
**Pull request by stephencelis**

### CODE REVIEW

File: Sources/Views/Collections/CollectionSection.swift [hunk 1]
```diff
@@ -112,48 +112,12 @@ private func coreLesson(
...
-      .gridColumn(
-        sizes: [.mobile: 3],
-        attributes: [
-          .class([
-            Class.grid.end(.mobile),
-          ]),
-        ],
-        .text(lesson.episode.length.formattedDescription)
-      )
+    contentRow(
```
Comment (stephencelis) [inline Sources/Views/Collections/CollectionSection.swift]: Sharing all row logic with a single helper now.

File: Sources/Styleguide/PointFreeBaseStyles.swift [hunk 2]
```diff
@@ -410,9 +409,12 @@ private let colorStyles: Stylesheet = concat([
 private let _codeClass = CssSelector.class("code")
 private let codeStyles =
   _codeClass % (
-    backgroundColor(.other("#fafafa"))
+//    backgroundColor(.other("#fafafa"))
```
Comment (mbrandonw) [inline Sources/Styleguide/PointFreeBaseStyles.swift]: should we delete this for now?

```suggestion
```

File: Sources/Styleguide/PointFreeBaseStyles.swift [hunk 3]
```diff
@@ -422,7 +424,8 @@ private let inlineCodeStyles =
       <> padding(topBottom: .px(1), leftRight: .px(5))
       <> borderWidth(all: .px(1))
       <> borderRadius(all: .px(3))
-      <> backgroundColor(Color.other("#fafafa"))
+//      <> backgroundColor(Color.other("#fafafa"))
```
Comment (mbrandonw) [inline Sources/Styleguide/PointFreeBaseStyles.swift]: this too?

```suggestion
```

File: Sources/Views/Markdown.swift [hunk 4]
```diff
@@ -56,8 +56,10 @@ private let codeMarkdownStyles: Stylesheet =
     )
     <> code % (
       fontFamily(["monospace"])
-        <> backgroundColor(.other("#fafafa"))
-        <> borderColor(all: Colors.gray850)
+//        <> backgroundColor(.other("#fafafa"))
+//        <> borderColor(all: Colors.gray850)
```
Comment (mbrandonw) [inline Sources/Views/Markdown.swift]: and these? 

```suggestion
```

**Pull request merged**